### PR TITLE
Upgrade chrome to 1108766

### LINKED ISSUE #2258: BrowserFetcher.DefaultChromiumRevision Exhibits Hang Bug When Page Has Odd Number column-count
### Description

From what we can tell Chromium has a bug where if the `column-count` is odd for this page (and possibly others), it'll thread spin hard and take _hours_ to complete a screenshot in some cases. Updating to `1115057` fixes it.

There is a block of code that allows us to replace `column-count: 3` with `column-count: 4`, this works, replacing it with 5 does not, 6 does, 7 does not, 8 does.

_Ideally_ moving up to a newer stable release may be ideal, I haven't been able to find a logged Chroium bug regarding this, it may have been silently fixed at some point.

### Complete minimal example reproducing the issue

See application here:

https://github.com/StrangeWill/PuppeteerSharpStall/blob/master/Program.cs

### Expected behavior:

A screenshot to be taken

### Actual behavior:

A stall and CPU spin for possibly _hours_ before screenshot is taken

### Versions

PuppeteerSharp 10.1.2

Edit:

We're just running with this version for now -- but I wanted visibility on this and a discussion around updating the default to prevent `column-count` related issues for anyone else.

### PR DESCRIPTION
**Pull request by kblok**
closes #2258

### CODE REVIEW

File: lib/PuppeteerSharp.Tests/NavigationTests/FrameGoToTests.cs [hunk 1]
```diff
@@ -41,7 +40,14 @@ public async Task ShouldRejectWhenFrameDetaches()
...
             var exception = await Assert.ThrowsAsync<NavigationException>(async () => await navigationTask);
-            Assert.Equal("Navigating frame was detached", exception.Message);
+            Assert.Contains(
+                "Navigating frame was detached",
+                new[]
+                {
+                    "Navigating frame was detached",
+                    "Error: NS_BINDING_ABORTED",
+                    "net::ERR_ABORTED",
+                });
```
Comment (jnyrup) [inline lib/PuppeteerSharp.Tests/NavigationTests/FrameGoToTests.cs:50]: Is this change intentional? The assertion is now always true as both subject and expectation are fixed values.

Comment (kblok) [reply lib/PuppeteerSharp.Tests/NavigationTests/FrameGoToTests.cs:50]: This matches the change [upstream](https://github.com/puppeteer/puppeteer/blob/d0c5b8e08905f3802705a1a90d7cc8fa04bc82db/test/src/navigation.spec.ts#L808) 🤷

Comment (jnyrup) [reply lib/PuppeteerSharp.Tests/NavigationTests/FrameGoToTests.cs:50]: No :)
The upstream asserts that `error.message` contains at least one of the three elements.
This change asserts that either of the three elements is `"Navigating frame was detached"`.

Comment (jnyrup) [reply lib/PuppeteerSharp.Tests/NavigationTests/FrameGoToTests.cs:50]: This is the best I can come up with xUnit
```cs
Assert.Contains(new[] { exception.Message }, message => new[]
{
    "Navigating frame was detached",
    "Error: NS_BINDING_ABORTED",
    "net::ERR_ABORTED"
}.Any(error => message.Contains(error)));
```

Comment (jnyrup) [reply lib/PuppeteerSharp.Tests/NavigationTests/FrameGoToTests.cs:50]: With FluentAssertions it can be written as
```cs
exception.Message.Should().ContainAny(
    "Navigating frame was detached",
    "Error: NS_BINDING_ABORTED",
    "net::ERR_ABORTED"
);
```

Comment (kblok) [reply lib/PuppeteerSharp.Tests/NavigationTests/FrameGoToTests.cs:50]: Oops you're so right 😅
The test passes right? 😅
